tests/foreman/ui/test_dashboard.py

```diff
@@ -0,0 +1,485 @@
+        3.Bookmark the search filter
+
+        @Assert: User is able to list the Bookmark
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_host_configuration_status(self):
+        """@Test: Review the Host Configuration Status widget
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Dashboard
+        2.Focus on the Host Configuration Status
+        3.Navigate to each of the links which has
+        search string associated with it.
+
+        @Assert: Each link shows the right info
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_host_configuration_chart(self):
+        """@Test: Check if Host Configuration Chart
+        widget is working in Dashboard UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Dashboard
+        2.Review the Host Configuration Chart widget
+        3.Navigate to each of the links which
+        has search string associated with it.
+
+        @Assert: Each link shows the right info
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_task_status(self):
+        """@Test: Check if Task Status Dashboard UI
+        is displayed
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Dashboard
+        2.Review the Task Status widget
+        3.Click each link
+
+        @Assert: Each link shows the right info
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_latest_warning_error_tasks(self):
+        """@Test: Check if Latest Warning/Error
+        Tasks widget is working in Dashboard UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Dashboard
+        2.Review the Latest Warning/Error Tasks widget.
+
+        @Assert: The links to all failed/warnings tasks are working
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_content_view_history(self):
+        """@Test: Check if Content View History
+           widget is working in Dashboard UI
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Dashboard
+        2.Review the Content View History widget
+
+        @Assert: Each Content View link shows its current status
+        (the environment to which it is published)
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_discovered_hosts(self):
+        """@Test: Check if user can access
+           Discovered Hosts widget
+
+        @Feature: Dashboard
+
+        @Steps:
+
+        1.Navigate to Monitor -> Dashboard
+        2.Review the Discovered Hosts widget
+        3.Click on the list of Discovered Hosts
+
+        @Assert: It takes you to discovered hosts
+
+        @Status: Manual
+        """
+
+    @stubbed()
+    @tier3
+    def test_positive_latest_event_widget(self):
```

Comment:
Please stick to the same name as UI - this should be `test_positive_latest_events_widget`